observation_portal/proposals/templates/proposals/proposal_detail.html

```diff
@@ -16,9 +16,22 @@ <h4>
         </h4>
         {% if user == object.pi %}
         <h4>Co Investigators <small><i id="toggleci" class="fa fa-eye"></i></small></h4>
-        <table class="table" id="cilist">
+        <div id="cilist">
+          {% if members_are_paginated %}
+          <form method="get" role="form" action="">
+            <div class="form-row">
+              <div class="col mb-0 mt-auto">{% bootstrap_field members_filter.form.first_name form_group_class="form-group mb-0 mt-auto" %}</div>
+              <div class="col mb-0 mt-auto">{% bootstrap_field members_filter.form.last_name form_group_class="form-group mb-0 mt-auto" %}</div>
+              <div class="col mb-0 mt-auto">{% bootstrap_field members_filter.form.username form_group_class="form-group mb-0 mt-auto" %}</div>
+              <div class="col mb-0 mt-auto">{% bootstrap_field members_filter.form.email form_group_class="form-group mb-0 mt-auto" %}</div>
```

Comment:
could probably just go with a single generic "filter" field that checks for that text in name, email, username. But this works too.

Reply:
Oh, do you have an example of how to do that? I was trying to do something like that but couldn't really figure out in a reasonable amount of effort a good way so just did this. I was able to easily get a single input to filter against many fields, but it was an AND or OR across those fields, and I didn't think that looping through each object in the queryset, and comparing against concatenated fields was a good idea. Maybe I didn't come across the right docs.